src/components/RuntimeManager.js

```diff
@@ -297,16 +298,16 @@ export default class RuntimeManager extends PureComponent {
           style: { display: 'flex' },
           disabled: appIsSettingUp(app),
           tooltip: appIsSettingUp(app) ?
-            'Your Galaxy application is being created' :
-            'Update cloud environment',
+            'Galaxy app is being created' :
+            'View cloud environment',
           onClick: () => {
             this.setState({ galaxyDrawerOpen: true })
           }
         }, [
           img({ src: galaxyLogo, alt: '', style: { marginRight: '0.25rem' } }),
           div([
             div({ style: { fontSize: 12, fontWeight: 'bold' } }, ['Galaxy']),
-            div({ style: { fontSize: 10, textTransform: 'uppercase' } }, [app.status])
+            div({ style: { fontSize: 10, textTransform: 'capitalize' } }, [_.capitalize(app.status)])
```

Comment:
I believe the use of the `textTransform` property in this div is duplicative of calling `_.capitalize` on the contents.

Reply:
Hmm...interestingly, `textTransform: 'capitalize'` doesn't seem to have any effect regardless of `_.capitalize(...)` so I took that out.